_data/changelogs/component-modal.yml

```diff
@@ -2,6 +2,11 @@ title: Modal
 type: component
 changelogURL:
 items:
+- date: NNNN-NN-NN
+  summary: Replaced deprecated `xlink:href` references with `href`.
+  affectsMarkup: true
+  githubPr: 6165
+  githubRepo: uswds
```

Comment:
```suggestion
  githubRepo: uswds
  versionUswds: N.N.N
```

Reply:
Resolved in [3ec6a6d](https://github.com/uswds/uswds-site/pull/2951/commits/3ec6a6de4ee0a7bf58b8a216a58233d2f7869d7e)